<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pomodoro</class>
 <widget class="QWidget" name="Pomodoro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1104</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>551</width>
     <height>511</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QToolButton" name="PomodoroSettings">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>30</y>
     <width>23</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>⚙️</string>
   </property>
  </widget>
  <widget class="QWidget" name="NotesWidget" native="true">
   <property name="geometry">
    <rect>
     <x>599</x>
     <y>30</y>
     <width>491</width>
     <height>531</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
